Cargo.toml

```diff
@@ -49,6 +49,7 @@ num-bigint = "0.4.6"
 pretty_assertions = "1.4.1"
 serde = { version = "1", features = ["serde_derive"] }
 serde_json = { version = "1" }
+smol_str = { version = "0.2", features = ["serde"] }
```

Comment:
FYI: cairo-lang-utils 2.11.2 does not reexport the `smol_str` so we have to use it directly here.